velocity/src/main/java/net/william278/papiproxybridge/user/VelocityUser.java

```diff
@@ -55,10 +56,10 @@ public UUID getUniqueId() {
     @Override
     public void sendPluginMessage(@NotNull PAPIProxyBridge plugin, @NotNull String channel, byte[] message) {
         player.getCurrentServer().ifPresent(server -> {
-            if (!server.sendPluginMessage(((VelocityPAPIProxyBridge) plugin).getChannelIdentifier(), message)) {
+            if (!server.sendPluginMessage(new LegacyChannelIdentifier(channel), message)) {
                 plugin.log(Level.SEVERE, "Failed to send plugin message to " + server.getServerInfo().getName()
                                          + " for player " + player.getUsername() + " on channel "
-                                         + ((VelocityPAPIProxyBridge) plugin).getChannelIdentifier().getId());
+                                         + channel);
```

Comment:
So the reason this is hardcoded like this afaik is because I think maybe on Fabric this breaks if it's like this? For this change would be good to run some tests with fabric as well as bungee (as I know those are slightly less common)

Reply:
I tested this with Fabric, Paper and Velocity, currently using it on production in my network, the only platform I don't have available to test is Bungee

I've encountered no issues with Fabric or Paper (paper doesn't use the components anyway)